Client/Frontend/Strings.swift

```diff
@@ -3837,16 +3837,18 @@ extension String {
             tableName: "Shopping",
             value: "Check Review Quality",
             comment: "Text for the analyzer button displayed when an analysis can be updated for a product.")
-        public static let NoAnalysisCardInProgressTitle = MZLocalizedString(
+        public static let NoAnalysisCardInProgressTitleNotUsed = MZLocalizedString(
             key: "Shopping.NoAnalysisCard.InProgress.Headline.Title.v120",
             tableName: "Shopping",
             value: "Checking review quality",
-            comment: "Title for the card displayed when a shopping product has not been analysed yet but the analysis is in progress.")
-        public static let NoAnalysisCardInProgressBodyLabel = MZLocalizedString(
+            comment: "Title for the card displayed when a shopping product has not been analysed yet but the analysis is in progress.",
+            lastUsedInVersion: 120)
+        public static let NoAnalysisCardInProgressBodyLabelNotUsed = MZLocalizedString(
             key: "Shopping.NoAnalysisCard.InProgress.Body.Label.v120",
             tableName: "Shopping",
             value: "This could take about 60 seconds.",
-            comment: "Description for the card displayed when a shopping product has not been analysed yet but the analysis is in progress.")
+            comment: "Description for the card displayed when a shopping product has not been analysed yet but the analysis is in progress.",
+            lastUsedInVersion: 120)
```

Comment:
Curious about this change? Does it tell the translation system to remove it?

Reply:
It's marks the string as not used so that we can remove it in a future version. If we'd remove it right now it would be removed from the translation system and that could cause issues. So we keep the string around until we can be sure most users are not on v120 anymore.

Reply:
[Wiki for reference](https://github.com/mozilla-mobile/firefox-ios/wiki/How-to-add-and-modify-Strings) 😄